### PR TITLE
[Exercise-e-shop][layout using scss][task] create basic layout using scss

### PR DESCRIPTION
**Pull request by st-suho**
**Description** 
*Create html documents and build layout for page https://www.figma.com/file/CUQiZbPKOlfjHZn9hBqrPh/Fashion?node-id=0%3A1
*Apply SCSS into excersie to make UI
**Dependence**
#7

### CODE REVIEW

File: exersise-e-shop/home.html [hunk 1]
```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/5d96d7f15a.js" crossorigin="anonymous"></script>
+  <title>Document</title>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:8]: change `title`

File: exersise-e-shop/home.html [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+      <div class="header-left">
+        <a href="#">
+          <img src="" alt="logo">
+        </a>
+      </div>
+      <div class="header-center">
+        <a href="#">Men</a>
+        <a href="#">Women</a>
+        <a href="#">Kids</a>
+      </div>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:22]: `html5` tag

File: exersise-e-shop/home.html [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+        <a href="#">
+          <i class="search-icon"></i>
+        </a>
+        <a href="#">
+          <i class="card-icon"></i>
+        </a>
+        <a href="#">
+          <i class="personal-icon"></i>
+        </a>
+      </div>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:33]: `ul > li`

File: exersise-e-shop/home.html [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+  </header>
+  <main>
+    <section class="banner">
+      <div class="summer-collection">
+        <h2 class="summer-title">
+          Sale of the <span>summer</span>collection
+        </h2>
+        <div class="shop-now">
+          <a href="#">
+            <i class="to-shop"></i>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:44]: lack of text `SHOP NOW`

File: exersise-e-shop/home.html [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+      <div class="details-clothes">
+        <img src="./photos/T-Shirt.png" alt="basic T-Shirt">
+        <h4>Basic t-shirts $29,99</h4>
+        <a href="#" class="btn">MORE DETAILS</a>
+      </div>
+      <div class="sale-off-clothes">
+        <img src="./photos/view-all.png" alt="sale this summer">
+        <h4>Sale this summer</h4>
+        <a href="#" class="btn">VIEW ALL</a>
+      </div>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:87]: `ul > li`

File: exersise-e-shop/home.html [hunk 6]
```diff
@@ -0,0 +1,303 @@
...
+    </section>
+    <section class="container recommended-clothes">
+      <h3>Selected just for you</h3>
+      <div>
+        <a href="#">SHOW MORE</a>
+      </div>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:93]: this action will show more info for this section, not section-header => should be absolute based on section

File: exersise-e-shop/home.html [hunk 7]
```diff
@@ -0,0 +1,303 @@
...
+      <ul>
+        <li>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:95]: define class name for this list

File: exersise-e-shop/home.html [hunk 8]
```diff
@@ -0,0 +1,303 @@
...
+            <h5>Loose Textured T-Shirt</h5>
+            <p>$119.99</p>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section class="advertisement-page">
+      <h3>Why should you choose us?</h3>
+      <ul>
+        <li>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:128]: define class name too

File: exersise-e-shop/home.html [hunk 9]
```diff
@@ -0,0 +1,303 @@
...
+          <div>
+            <img src="./photos/loose-tshirt.png" alt="Textured T-Shirt">
+            <h5>Loose Textured T-Shirt</h5>
+            <p>$119.99</p>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:193]: need a class name to styling `img/background`

File: exersise-e-shop/home.html [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+      <div class="subscribe-page">
+        <p>Subscribe to our newsletter andreceive exclusive offers every week</p>
+        <div class="enter-email">
+          <input class="email-form" placeholder="Enter your email" type="email">
+        </div>
+        <button class="subscribe">
+          SUBSCRIBE
+        </button>
+      </div>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:202]: need container too

File: exersise-e-shop/home.html [hunk 11]
```diff
@@ -0,0 +1,303 @@
...
+            House My Brand designs clothing for the young, the old &everyone in between – but most importantly, for the
+            fashionable
+          </p>
+          <div class="social-icon">
+            <i class="fab fa-facebook-f icon"></i>
+            <i class="fab fa-twitter icon"></i>
+            <i class="fab fa-invision icon"></i>
+            <i class="fab fa-instagram-square icon"></i>
+            <i class="fab fa-youtube icon"></i>
+          </div>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:220]: `ul > li`

File: exersise-e-shop/home.html [hunk 12]
```diff
@@ -0,0 +1,303 @@
...
+        </div>
+      </div>
+      <div class="footer-bottom">
+        DESIGN BY ICEO.CO - &copy; 2019. ALL RIGHTS RESERVED.
+      </div>
+    </div>
+  </footer>
+</body>
+
+</html>
```
Comment (st-quocpham) [inline exersise-e-shop/home.html:303]: lack of end line

File: exersise-e-shop/home.html [hunk 13]
```diff
@@ -0,0 +1,303 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/5d96d7f15a.js" crossorigin="anonymous"></script>
+  <title>Document</title>
+</head>
+<body>
+  <header class="page-header">
+    <div class="container wrap-header">
+      <div class="header-left">
```
Comment (st-hahoang) [inline exersise-e-shop/home.html:13]: Trang top nên có h1 ở logo.

File: exersise-e-shop/stylesheet/base/_mixin.scss [hunk 14]
```diff
@@ -0,0 +1,22 @@
+.grid-product {
```
Comment (st-quocpham) [inline exersise-e-shop/stylesheet/base/_mixin.scss:1]: it's not a `mixin`

**Pull request merged**